ignite/evaluator.py

```diff
@@ -0,0 +1,84 @@
+import time
+
+from torch.autograd import Variable
+
+from ignite.history import History
+from ignite.engine import Engine
+
+from enum import Enum
+
+
+class EvaluationEvents(Enum):
+    EVALUATION_STARTING = "evaluation_starting"
+    EVALUATION_COMPLETED = "evaluation_completed"
+    EVALUATION_ITERATION_STARTED = "evaluation_iteration_started"
+    EVALUATION_ITERATION_COMPLETED = "evaluation_iteration_completed"
+    EXCEPTION_RAISED = "exception_raised"
+
+
+def _to_hours_mins_secs(time_taken):
+    mins, secs = divmod(time_taken, 60)
+    hours, mins = divmod(mins, 60)
+    return hours, mins, secs
```

Comment:
This code is duplicated in `trainer.py`

Reply:
yeah - I should put this in a utils somewhere